src/app/member/components/MemberActionRender.tsx

```diff
@@ -40,14 +48,14 @@ const MemberTableActionRender = ({ role = 'shift_incharge', status = 'active' }:
 
     const actionMenu = React.useMemo(() => {
       switch (role) {
-        case 'member':
+        case MemberRole.Member:
           return status === 'inactive' 
```

Comment:
can we create status enums and use here instead of string value

Reply:
@ghulamhafeez  please review here these enum status